lib/src/values/logic_value.dart

```diff
@@ -761,6 +768,35 @@ abstract class LogicValue {
       // ignore: avoid_dynamic_calls
       _doCompare(other, (a, b) => (a >= b) as bool);
 
+  /// Binary Power
+  ///
+  /// Both inputs [a] and [b] are either of type [int] or [BigInt].
+  /// Returns [a] raise to the power [b] of input type.
+  dynamic _binPower(dynamic a, dynamic b) {
```

Comment:
can we leverage built-in `pow` methods for `int` and `BigInt` since they are more efficient and already well-tested?
https://api.dart.dev/stable/2.19.6/dart-math/pow.html
https://api.flutter.dev/flutter/dart-core/BigInt/pow.html